### DISCUSSION TITLE
umi 打包报错

### DISCUSSION THREAD

**Discussion by jockiech** [umijs/umi#11717] [Q&A]
![image](https://github.com/umijs/umi/assets/38551182/2294b20c-9b90-48f0-837b-6e433f40d364)
在打包时遇到这个报错，不知道为什么，umi的版本是4.0.83，antd-pro是最新的版本，然后webpack是5

**Comment by gelingyan** [comment]
把umi的版本固定，"@umijs/max": "4.0.57",

**Comment by fz6m** [comment]
删一下 `node_modules/.cache` 缓存之后重新启动就可以了，我测试最新的 umi 版本 `4.0.83` 导入 `dayjs/plugin/customParseFormat` 没问题。
